.github/workflows/pr.yml

```diff
@@ -6,53 +6,111 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [16]
+        node:
+          - 20
     steps:
       - name: Checkout repo
         uses: actions/checkout@v3
-      - name: Use Node ${{ matrix.node }}
+      - name: "Use Node ${{ matrix.node }}"
         uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node }}
-          cache: "yarn"
+          node-version: "${{ matrix.node }}"
+          cache: yarn
+      - name: Setup Anchor & Build
+        uses: metadaoproject/setup-anchor@v2
+        with:
+          anchor-version: 0.30.1
+          solana-cli-version: 1.18.21
+          node-version: 20.17.0
+      - name: Create keypair
+        run: solana-keygen new --no-bip39-passphrase
+        shell: bash
+      - name: Make Anchor.toml compatible with runner
+        run: "sed -i 's:/user/:/runner/:' Anchor.toml"
```

Comment:
I don't see the string `user` in the Anchor.toml?

Reply:
I might actually be able to delete this. it was trying to get anchor to detect the `solana-keygen new` command from above. the util can in some environments place the key in a location anchor cant find it, hence this command. but it looks like we don't need that after all and the tests run without it.